sql/core/src/main/scala/org/apache/spark/sql/avro/SchemaConverters.scala

```diff
@@ -23,13 +23,12 @@ import scala.collection.mutable
 import scala.jdk.CollectionConverters._
 
 import org.apache.avro.{LogicalTypes, Schema, SchemaBuilder}
-import org.apache.avro.LogicalTypes.{Date, Decimal, LocalTimestampMicros, LocalTimestampMillis, TimestampMicros, TimestampMillis}
+import org.apache.avro.LogicalTypes.{Decimal, _}
```

Comment:
why not just do 

```
import org.apache.avro.LogicalTypes._
```

Reply:
This doesn't work, because we need Decimal.___